modules/kml/src/gpx-loader.ts

```diff
@@ -1,4 +1,5 @@
 import {geojsonToBinary} from '@loaders.gl/gis';
+import {LoaderObject} from '@loaders.gl/loader-utils/types';
```

Comment:
Use `import type {` when importing pure types (among other things this avoids problems when we transpile to js with babel).